org.testeditor.web.backend.persistence/build.gradle

```diff
@@ -20,3 +20,18 @@ dependencyCheck {
     failBuildOnCVSS=11
     suppressionFile=file("suppressions.xml").getCanonicalPath()
 }
+
+task docker(type:Exec) {
+    dependsOn(build)
+    executable 'sh'
+    args '-c', """
+      (([ -d testeditor ] && rm -Rf testeditor) || true) &&
+      cd build/distributions && (rm persistence.tar || true) &&
+      rm org.testeditor.*.zip &&
+      mv *.tar persistence.tar &&
```

Comment:
I guess this breaks if there is more than one tar file in the directory, so should this be more specific?